### PR TITLE
chore: Upgrade material-ui to 4.3.3

### PR DESCRIPTION
**Pull request by cezres**
Older version of material-ui has an error during build.

屏幕快照2019-08-22上午11 55 17

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,5 +11,8 @@ install:
   # - yarn bootstrap
   - travis_retry make ci-install
 script:
+  - export SERVER=https://node.cryptape.com
```
Comment (rainchen) [inline .travis.yml]: should  not use testnet for testing in CI

Comment (cezres) [reply .travis.yml]: Ok, what other nodes can I connect to? Or skip this test?

Comment (rainchen) [reply .travis.yml]: @Keith-CY how to do system testing before?

Comment (Keith-CY) [reply .travis.yml]: All tests run locally by the `pre-commit` hook and only the tests of `utils`, `formatters`, `address` modules run in the CI.

Comment (Keith-CY) [reply .travis.yml]: We've planned to run a docker in the CI, but it's suspended.

Comment (cezres) [reply .travis.yml]: @rainchen  We configure the CI environment parameters at https://travis-ci.org.
A commit with a CI error does not seem to read the environment parameter, PRIVATE_KEY is undefined.

Comment (rainchen) [reply .travis.yml]: using ENV in travisCI is an acceptable solution before we can have an auto testing env with docker

**Pull request closed without merging**